js/src/index.js

```diff
@@ -98,7 +95,7 @@ if (module.hot) {
   //   //   <AppContainer>
   //   //     <ContextProvider api={ api } muiTheme={ muiTheme } store={ newStore }>
   //   //       <MainApplication
-  //   //         routerHistory={ routerHistory }
+  //   //         routerHistory={ hashHistory }
```

Comment:
We sure can. It's just that right now whenever the Redux store is modified, the whole app has to refresh. With a bit of work, this could be hot reloaded